tests/csapi/ignored_users_test.go

```diff
@@ -0,0 +1,105 @@
+// +build !dendrite_blacklist
+
+// Rationale for being included in Dendrite's blacklist: https://github.com/matrix-org/dendrite/issues/600
+package csapi_tests
+
+import (
+	"net/url"
+	"testing"
+
+	"github.com/tidwall/gjson"
+
+	"github.com/matrix-org/complement/internal/b"
+	"github.com/matrix-org/complement/internal/client"
+	"github.com/matrix-org/complement/internal/match"
+	"github.com/matrix-org/complement/internal/must"
+)
+
+// The Spec says here
+//     https://spec.matrix.org/v1.1/client-server-api/#server-behaviour-13
+// that
+// > Servers must not send room invites from ignored users to clients.
+//
+// Synapse does not have this property, as detailed in
+// https://github.com/matrix-org/synapse/issues/11506.
+// This reproduces that bug.
+func TestInviteFromIgnoredUsersDoesNotAppearInSync(t *testing.T) {
+	deployment := Deploy(t, b.BlueprintCleanHS)
+	defer deployment.Destroy(t)
+	alice := deployment.RegisterUser(t, "hs1", "alice", "sufficiently_long_password_alice")
+	bob := deployment.RegisterUser(t, "hs1", "bob", "sufficiently_long_password_bob")
+	chris := deployment.RegisterUser(t, "hs1", "chris", "sufficiently_long_password_chris")
+
+	// Alice creates a room for herself.
+	public_room := alice.CreateRoom(t, map[string]interface{}{
```

Comment:
Please do not use underscores for variable names.

Reply:
Sorry, old habits. Is there some way to configure `gofmt`/`goimports` or VSCode to flag this up?